src/TelegramHandler.php

```diff
@@ -8,26 +8,69 @@
 use Psr\Http\Client\ClientInterface;
 use Psr\Http\Message\RequestFactoryInterface;
 use Psr\Http\Message\RequestInterface;
+use InvalidArgumentException;
 
 /**
- * Logs to Cube.
- *
- * @link http://square.github.com/cube/
+ * Logs to Telegram.
  */
 final class TelegramHandler extends AbstractHttpClientHandler
 {
+    private const TELEGRAM_API = 'https://api.telegram.org/bot';
+
+    /**
+     * @var string
+     */
+    private $apiKey;
+
+    /**
+     * @var int|string
+     */
+    private $chatId;
+
+    /**
+     * @param string $apiKey
+     * @param int|string $chatId
+     * @param ClientInterface $client
+     * @param RequestFactoryInterface $requestFactory
+     * @param int $level
+     * @param bool $bubble
+     */
     public function __construct(
+        string $apiKey,
+        $chatId,
         ClientInterface $client,
         RequestFactoryInterface $requestFactory,
         $level = Logger::ERROR,
         bool $bubble = true
     ) {
+        $this->apiKey = $apiKey;
+        $this->chatId = $chatId;
         parent::__construct($client, $requestFactory, $level, $bubble);
     }
 
-    // todo
+    /**
+     * Create request to @link https://api.telegram.org/bot on SendMessage action
+     * @see https://core.telegram.org/bots/api#sendmessage
+     *
+     * @param array $record
+     * @return RequestInterface
```

Comment:
Let's remove superflous docblocks.

There is also a php-cs-fixer rule: https://mlocati.github.io/php-cs-fixer-configurator/#version:2.16|fixer:no_superfluous_phpdoc_tags

maybe we should include this rule.. can you add it?